doc/source/tutorial/stats/sampling.rst

```diff
@@ -0,0 +1,202 @@
+samples from the given distribution.
+
+An example of this interface is shown below:
+
+    >>> from scipy.stats import TransformedDensityRejection
+    >>> from math import exp
+    >>> 
+    >>> class StandardNormal:
+    ...     def pdf(self, x: float) -> float:
+    ...         # note that the normalization constant isn't required
+    ...         return exp(-0.5 * x*x)
+    ...     def dpdf(self, x: float) -> float:
+    ...         return -x * exp(-0.5 * x*x)
+    ... 
+    >>> dist = StandardNormal()
+    >>> 
+    >>> urng = np.random.default_rng(0x6be00a9336fc82258fe73bfa672d3d52)
+    >>> rng = TransformedDensityRejection(dist, seed=urng)
+
+As shown in the example, we first initialize a distribution object that
+contains an implementation of the methods required by the generator. In
+our case, we use the :class:`~TransformedDensityRejection` method which
+requires a PDF and its derivative w.r.t. x (i.e. the variate).
+
+.. note:: Note that the methods of the distribution (i.e. ``pdf``,
+          ``dpdf``, etc) need not be vectorized. They should
+          accept and return floats.
+
+.. note:: One can also pass the SciPy distributions as arguments but it can
+          be slow due to validations and expensive NumPy operations.
+          Moreover, it doesn't always have all the information required
+          by some generators like derivative of PDF for TDR method. Also,
+          most of the distributions in SciPy provide a ``rvs`` method which
+          can be used instead.
+
+In the above example, we have set up an object of the
+:class:`~TransformedDensityRejection` method to sample from a
+standard normal distribution. Now, we can start sampling from our
+distribution by calling the ``rvs`` method:
+
+    >>> rng.rvs()
+    -0.6853213192313732
+    >>> rng.rvs((5, 3))
+    array([[ 0.24129542,  1.28709556,  0.43244247],
+           [ 0.88977668,  1.22981663, -0.09820373],
+           [ 1.38588147, -1.06629264,  0.54475257],
+           [-0.55153436, -0.67281406,  0.01830143],
+           [ 1.37709649, -1.33036774,  0.62131967]])
+
+.. note:: Please note the difference between the `rvs` method of the
+          distributions present in :mod:`scipy.stats` and the one provided
+          by these generators. In general, the aim of the generation
+          methods implemented in :class:`~rv_continuous` is to provide
+          generators that work reasonably well for common use cases, e.g.
+          to generate small samples using different shape parameters or to
+          generate large samples for a fixed shape parameter. Depending on
+          the goal, specialized methods can lead to a substantial speedup.
+          For example, at the cost of a rather expensive setup step, very
+          fast generation can be achieved for large samples and a fixed
+          shape parameter. Also, even if the same URNG (``seed``) is used,
+          the resulting rvs will be different in general.
```

Comment:
```suggestion
          the resulting ``rvs`` will be different in general.
```
Why is that the case? It should say, just a line.

Reply:
> Why is that the case?

This difference exists between the `rvs` method of distributions in SciPy and the `rvs` method of UNU.RAN generators. It is because UNU.RAN transforms uniform random numbers differently than SciPy of NumPy. Is this not clear enough? Should I make this more explicit?

Reply:
What I don't get with this is: If you say that having the same `seed=urng` does not output the same `rvs`, then why do we have this parameter?

Reply:
> If you say that having the same `seed=urng` does not output the same `rvs`, then why do we have this parameter?

For instance, if we set `seed=urng`, the random number stream will match for the same method (say, `TransformedDensityRejection`).  But the rvs stream of SciPy's `norm` and UNU.RAN's `TransformedDensityRejection` would be the same even for the same `seed`/`random_state`:

```python
>>> from scipy.stats import norm, TransformedDensityRejection
>>> 
>>> class StdNorm:
...     def pdf(self, x):
...         return np.exp(-0.5 * x*x)
...     def dpdf(self, x):
...         return -x * self.pdf(x)
... 
>>> dist = StdNorm()
>>> rng = TransformedDensityRejection(dist, seed=123)
>>> rng.rvs()
0.474548717355228
>>> norm.rvs(random_state=123)
-1.0856306033005612
```

Moreover, different methods also produce different streams of RVs. So, for example, for the same `seed`, the output of `TransformedDensityRejection` would differ from `NumericalInverseHermite`. Hope this helps!

Reply:
Ah ok, I thought you were talking about UNU.RAN only here. The `URNG` was misleading I think.

Reply:
I have made this clearer now. (see 3dcd2689a46ffdac2ff57a949b46d9a165c55d83)